decompiler/special.py

```diff
@@ -0,0 +1,103 @@
+# Separated from unrpyc.py
```

Comment:
Maybe rename this file to something more descriptive. Like `renpycompat.py`

It'd be nice to be a bit more descriptive in the top comment as well, `# Separated from unrpyc.py` provides very little information to anyone that's not reading this specific commit.

I like the idea of splitting this out. It means there'll be less duplication with `un.rpyc`. And it can simplify the dependency between files a bit.

Reply:
Naming(good one) and description is right. Will do. Not sure however if i come up with a good desc...

> less duplication ... simplify the dependency

That was one point of this. Another is, i wanted to get some breathing space in the main file and do also feel this code belongs not really there. I thought also about adding this to until.py.